### PR TITLE
Add dashboard navigator

### LINKED ISSUE #87: Document navigator
Document the Navigator feature in the dashboard:
image

### PR DESCRIPTION
**Pull request by karolkielecki**
Fix #87

### CODE REVIEW

File: docs/dashboard/home.md [hunk 1]
```diff
@@ -60,4 +60,14 @@ If you prefer to work in dark mode, click on your account name in the top right
 
 ### How to log out
 
-To log out, click your account name in the top right corner of the dashboard, and then click the _log out_ button. 
+To log out, click your account name in the top right corner of the dashboard, and then click the _log out_ button.
```
Comment (krzysztofwolski) [inline docs/dashboard/home.md:63]: image

Available actions should be described here

**Review by krzysztofwolski — CHANGES_REQUESTED** [review]

File: docs/dashboard/home.md (hunk 1 above)
Comment (krzysztofwolski) [reply docs/dashboard/home.md:63]: Comment should be to navigator section description.

File: docs/dashboard/home.md [hunk 2]
```diff
@@ -60,4 +60,22 @@ If you prefer to work in dark mode, click on your account name in the top right
...
+
+### How to use the dashboard navigator
+
+Click the navigator button on top right corner of the dashboard or use the *CTRL* + *K* combination to open the dashboard navigator. Then type the location you would like to navigate to, for instance: products main page.
+
+![Dashboard navigator](screenshots/dashboard-home-navigator.png)
+
+You can type "?" to see all available navigation options.
+
+Available navigation options:
```
Comment (krzysztofwolski) [inline docs/dashboard/home.md:73]: This description doesn't add any explanation on how to use it other than doubling text which user already can see.

**Review by krzysztofwolski — CHANGES_REQUESTED** [review]

File: docs/dashboard/home.md (hunk 2 above)
Comment (karolkielecki) [reply docs/dashboard/home.md:73]: Ok, I will add an explanation on how to use it.

**Review by krzysztofwolski — APPROVED** [review]
👍

**Review by maarcingebala — APPROVED** [review]

**Pull request merged**